src/commands/execute.rs

```diff
@@ -0,0 +1,346 @@
+//! Module for wax, executes a module immediately
+
+use crate::data::wax_index;
+use crate::dataflow::find_command_result::FindCommandResult;
+use crate::dataflow::installed_packages::{InstalledPackages, RegistryInstaller};
+use crate::dataflow::lockfile_packages::LockfileResult;
+use crate::dataflow::resolved_packages::ResolvedPackages;
+use crate::dataflow::WapmPackageKey;
+use crate::graphql::{execute_query, DateTime};
+use crate::keys;
+use crate::util;
+
+use graphql_client::*;
+
+use std::convert::From;
+use std::env;
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::str::FromStr;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+pub struct ExecuteOpt {
+    /// Command name
+    command: String,
+    /// Run unsandboxed emscripten modules too
+    #[structopt(long = "emscripten")]
+    run_emscripten_too: bool,
+    /// Agree to all prompts. Useful for non-interactive uses. (WARNING: this may cause undesired behavior)
+    #[structopt(long = "force-yes", short = "y")]
+    force_yes: bool,
+    /// WASI pre-opened directory
+    #[structopt(long = "dir", multiple = true, group = "wasi")]
+    pre_opened_directories: Vec<String>,
+    /// Print info about Wax instead of executing a command
+    #[structopt(long = "info")]
+    just_print_info: bool,
+    /// Application arguments
+    #[structopt(raw(multiple = "true"), parse(from_os_str))]
+    args: Vec<OsString>,
+}
+
+#[derive(Debug, Fail)]
+enum ExecuteError {
+    #[fail(
+        display = "No package for command `{}` found locally or in the registry",
```

Comment:
```suggestion
        display = "No package for command `{}` in the registry or in local packages",
```

I know we search first locally and then in the registry, but I think it makes more sense to mention the other way to the user. Thoughts?

Reply:
Sure, I have no strong opinion about this wording here so will defer to you!